<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventsTab</class>
 <widget class="QWidget" name="EventsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="labelEventDelay">
         <property name="toolTip">
          <string>Delay applied before sending automatically generated events</string>
         </property>
         <property name="text">
          <string>Delay for asynchronous events</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinEventDelay">
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>3600000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Address related errors</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editAddressErrors">
            <property name="text">
             <string>dummy.user@proton.me</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QPushButton" name="buttonAddressChanged">
            <property name="text">
             <string>Address Changed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonAddressChangedLogout">
            <property name="text">
             <string>Address Changed Logout</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkNextCacheChangeWillSucceed">
         <property name="text">
          <string>Next Cache Change will succeed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkIsPortFree">
         <property name="text">
          <string>Reply true to the next 'Is Port Free' request.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="labelNextBugReportResult">
         <property name="text">
          <string>Next bug report result</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBugReportResult">
         <item>
          <property name="text">
           <string>Success</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Error</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Data sharing error</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="verticalSpacing">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonUpdateManualReady">
         <property name="text">
          <string>Update Manual Ready</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="editUpdateVersion">
         <property name="text">
          <string>4.0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="buttonUpdateVersionChanged">
         <property name="text">
          <string>Update version changed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonUpdateForce">
         <property name="text">
          <string>Update Force</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="buttonUpdateManualRestart">
         <property name="text">
          <string>Update manual restart</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="buttonUpdateCheckFinished">
         <property name="text">
          <string>Update check finished</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="buttonUpdateSilentRestart">
         <property name="text">
          <string>Update silent restart</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="comboUpdateError">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Update manual error</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Update force error</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Update silent error</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="buttonUpdateError">
         <property name="text">
          <string>Update error</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="buttonUpdateIsLatest">
         <property name="text">
          <string>Update is latest</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="horizontalSpacing">
        <number>-1</number>
       </property>
       <property name="verticalSpacing">
        <number>8</number>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonInternetOn">
         <property name="text">
          <string>Internet On</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="buttonShowMainWindow">
         <property name="text">
          <string>Show Main Window</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonInternetOff">
         <property name="text">
          <string>Internet Off</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="buttonAPICertIssue">
         <property name="text">
          <string>API Certficate Issue</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="buttonNoKeychain">
         <property name="text">
          <string>No Keychain</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>200</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
